ipi/engine/thermostats.py

```diff
@@ -194,7 +194,104 @@ def step(self):
 
       self.p = p
       self.ethermo = et
-      
+
+
+class ThermoNFL(Thermostat):
+   """Represents a langevin thermostat for systems driven by statistical (noisy) forces,
+      adding adequate additional damping for the inherent white noise term from the forces.
+      If sigtau is set > 0, sigma will be automatically adjusted over the time until the
+      system reaches the target temperature.
+
+   Depend objects:
+      tau: Thermostat damping time scale. Larger values give a less strongly
+         coupled thermostat.
+      sigma: Force noise amplitude sigma. Larger sigma means larger additional damping term.
+      sigtau: Sigma-Temperature coupling time constant tau.
+      T: Coefficient of the diffusive contribution of the thermostat, i.e. the
+         drift back towards equilibrium. Depends on tau and the time step.
+      S: Coefficient of the stochastic contribution of the thermostat, i.e.
+         the uncorrelated Gaussian noise. Depends on T and the temperature.
+   """
+
+   def get_T(self):
+      """Calculates the coefficient of the overall drift of the velocities."""
+
+      SigT = self.sigma**2/(Constants.kb*self.temp)
```

Comment:
Are you sure about this line? This does not make dimensional sense, T is a pure number, sigma is (according to the input definition) a force, and kT is an energy.